configobject/statesync/statesync.go

```diff
@@ -53,7 +53,7 @@ func logSyncCounters() {
 	for {
 		<-every20s.C
 		if syncCounter["host"] > 0 || syncCounter["service"] > 0 {
-			log.Infof("Synced %d host and %d service states in the last 20 seconds", syncCounter["host"], syncCounter["service"])
+			log.WithFields(log.Fields{"sync_counter_host" : syncCounter["host"], "sync_counter_service" : syncCounter["service"]}).Info("Synced host and service states in the last 20 seconds")
```

Comment:
* "Synced host" => "Synced some host"
* "sync_counter_host" => "host_states" and the same for the service